crates/pathfinder/src/consensus/inner/p2p_task.rs

```diff
@@ -227,6 +250,14 @@ pub fn spawn(
                             assert_ne!(*proposer, Address(proposal.proposer.0));
                             *round = proposal.round.as_u32().expect("Round not to be None");
                             *proposer = Address(proposal.proposer.0);
+                            let proposer_address = ContractAddress(proposal.proposer.0);
+                            persist_proposal_parts(
+                                db_tx,
+                                proposal.height,
+                                *round,
+                                &proposer_address,
+                                &proposal_parts,
+                            )?;
```

Comment:
and ends here.

Reply:
https://github.com/eqlabs/pathfinder/commit/741026acdc911e0fa5d5f6d041adb3758b4f2d73